### ISSUE TITLE
SCSS: classes pulled in via @use are reported as non-existent

### ISSUE THREAD

**Issue by mattrussell-sonocent** [malinindev/check-unused-css#90]
Checked on version 0.5.2

## Example

`src/_shared.scss`:

```scss
.box {
  color: red;
}
```

`src/Foo.module.scss`:

```scss
@use 'shared';
```

`src/Foo.tsx`:

```tsx
import styles from './Foo.module.scss'

export const Foo = () => <div className={styles.box} />
```

Running `check-unused-css src`:

```
Found 1 classes used in source files but non-existent in CSS:
  Foo.tsx:3:49 - .box
```

**Comment by malinindev** [comment]
Fixed in v0.5.3
